src/Core/Framework/Adapter/Twig/StringTemplateRenderer.php

```diff
@@ -15,14 +15,25 @@ class StringTemplateRenderer
      */
     private $twig;
 
+    /**
+     * @var Environment
+     */
+    private $platformTwig;
+
     public function __construct(Environment $environment)
+    {
+        $this->platformTwig = $environment;
+        $this->initialize();
```

Comment:
How long does this `initialize` take time. This will be now always called when this service will be injected

Reply:
the initialize code is now only out-sourced into a method. it was in the constructor before, too :smile: